configure.ac

```diff
@@ -44,59 +58,122 @@ AC_PROG_CPP
 
 # Find the Fortran compiler.
 AC_PROG_FC
-AC_PROG_F77
 
 # Find the install program.
 AC_PROG_INSTALL
 
 # Is bats installed? It is required for testing.
-AC_CHECK_PROGS([FMS_BATS], [bats])
-if test -z "$FMS_BATS"; then
-   AC_MSG_ERROR([Cannot find bats utility. Install bats and try again.])
+AC_CHECK_PROGS([BATS], [bats])
+AM_CONDITIONAL([MAKE_CHECK_ENABLED], [test -n "$BATS"])
+if test -z "$BATS"; then
+   AC_MSG_WARN([Cannot find the bats utility. All checks will be disabled.])
```

Comment:
This is a poor idea. You are encouraging users to build your package without running tests.

Is the minor benefit of bats worth that?

Reply:
We still have some of our systems that do not have bats, and I don't want our users (who adopt the autotools build) to not be able to use this build method, which is why I decided to allow this.

I am investigating how to deal with the tests w/o bats, and if possible keep the TAP output. When we decide on how to do the testing w/o bats, we will remove this dependency.

Reply:
OK, how about we change autoconf right now to warn you that make check will not work without bats. (But make all still will work of course).

Then, machines that don't have bats will not be able to run tests, but will still be able to build the code.

We previously discussed making bats optional. The more you have to distort your build to support bats, the less cost-effective it is. How about we change to use bats optionally, that is, use it when present, and don't when not, and determine whether it is present in configure?

So instead of checking in and maintaining these changes, how about we make bats optional? This can be done when I get back from vacation, and meanwhile make check will not work, but make all will.